helmclient.go

```diff
@@ -394,7 +394,16 @@ func (c *Client) InstallFromTarball(path, ns string, options ...helmclient.Insta
 		release, err := c.newHelmClientFromTunnel(t).InstallRelease(path, ns, options...)
 		if IsCannotReuseRelease(err) {
```

Comment:
I would also write a test for that matches but I don't understand when this happens.